esx_service/cli/vmdkops_admin.py

```diff
@@ -420,8 +420,8 @@ def commands():
                     'help': 'Volume to set options for, specified as "volume@datastore".',
                     'required': True
                 },
-                '--tenant': {
-                    'help': 'Name of the tenant the volume belongs to.',
+                '--vm-group': {
```

Comment:
Talked with @shuklanirdesh82 , upgrade the photon OS version. Then rerun test "make test-all", it failed only one test "TestConcurrency", which is a known issue.